chia/cmds/wallet_funcs.py

```diff
@@ -308,21 +310,37 @@ def timestamp_to_time(timestamp):
     return datetime.fromtimestamp(timestamp).strftime("%Y-%m-%d %H:%M:%S")
 
 
-async def print_offer_summary(wallet_client: WalletRpcClient, sum_dict: dict):
+async def print_offer_summary(cat_name_resolver: CATNameResolver, sum_dict: Dict[str, int], has_fee: bool = False):
     for asset_id, amount in sum_dict.items():
-        if asset_id == "xch":
-            wid: str = "1"
-            name: str = "XCH"
-            unit: int = units["chia"]
-        else:
-            result = await wallet_client.cat_asset_id_to_name(bytes32.from_hexstr(asset_id))
-            wid = "Unknown"
+        description: str = ""
+        unit: int = units["chia"]
+        wid: str = "1" if asset_id == "xch" else ""
+        mojo_amount: int = int(Decimal(amount))
+        name: str = "XCH"
+        if asset_id != "xch":
             name = asset_id
-            unit = units["cat"]
-            if result is not None:
-                wid = str(result[0])
-                name = result[1]
-        print(f"    - {name} (Wallet ID: {wid}): {Decimal(int(amount)) / unit} ({int(Decimal(amount))} mojos)")
+            if asset_id == "unknown":
```

Comment:
This is the new bit of handling for "unknown" outbound balances. We believe the amount is likely to be the returned change associated with the included fee coin(s), but we aren't 100% sure, hence the description set below.